src/common/utils/ldap/ldap.go

```diff
@@ -31,77 +30,70 @@ import (
 	goldap "gopkg.in/ldap.v2"
 )
 
-// GetSystemLdapConf ...
-func GetSystemLdapConf() (models.LdapConf, error) {
+//Session - define a LDAP session
+type Session struct {
+	ldapConfig models.LdapConf
+	ldapConn   *goldap.Conn
+}
+
+//LoadSystemLdapConfig - load LDAP configure from adminserver
+func LoadSystemLdapConfig() (*Session, error) {
```

Comment:
Consider rename this func to CreateSessionWithConfig?

Reply:
createWithInternalConfig
LoadSystemLdapConfig
CreateWithUIConfig

These three functions are created by UI use different values, issue #3763 is created to eliminate ambiguous functions